specification/metadata/metadata_structure.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
```

Comment:
I would move some of this to the mdpp since otherwise when stitched together, these all get nested (there's a brief description of metadata in that file and also here).